pkg/api/mlflow/dao/models/metric.go

```diff
@@ -11,6 +11,11 @@ type Metric struct {
 	Iter      int64   `gorm:"index"`
 }
 
+// TableName returns actual table name.
+func (p Metric) TableName() string {
+	return "metrics"
+}
+
```

Comment:
I have seen in other PRs that we introduced these `TableName` methods. Out of curiosity, what is the reasoning behind them?

Reply:
ahh, yep. This is `tabler` interface and it needs to run this logic:
```
for _, table := range []schema.Tabler{
		models.Tag{},
		models.Param{},
		models.LatestMetric{},
		models.Metric{},
		models.Run{},
		models.ExperimentTag{},
		models.Experiment{},
	} {
		if err := f.db.Session(&gorm.Session{AllowGlobalUpdate: true}).Delete(table).Error; err != nil {
			return errors.Wrapf(err, "error deleting data from %s table", table.TableName())
		}
	}
```

Reply:
hmmm, I am pretty sure that there is a default naming method in Gorm that will give you the name instead

Reply:
it is a pretty common practise. I can't see any problem to follow Gorm standards.

Reply:
gorm could generate it somewhere internally - yes, but I prefer to control it and use for logging as well like:
```
return errors.Wrapf(err, "error deleting data from %s table", table.TableName())
```

Reply:
so this is only used for logging in tests — I would argue that we shouldn't define all of this code just for that, especially as the `err` returned by Gorm is already "no such table: runs", so simply having `return errors.Wrapf(err, "error unloading fixtures")` would give us all the information we need

Reply:
you are strong fighter =). ok let me check.

Reply:
done.